core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala

```diff
@@ -61,6 +61,7 @@ object Main extends {
       checkCRD(settings, client)
 
       val ownerReferences = getDeploymentOwnerReferences(settings, client)
+      installProtocolVersion(settings, client, ownerReferences)
```

Comment:
What about adding this to helm chart?

Reply:
I guess the cloudflow currently already has permissions to create secrets in its own namespace, so is fine like this too. But feels a bit like a helm thing. (I'm ok what is done here, just a comment)

Reply:
make sense, I just restored the old way of doing it, for now, doing it in code makes us faster to change it, but helm makes sense too.